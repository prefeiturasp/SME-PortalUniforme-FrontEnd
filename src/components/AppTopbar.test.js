import { mount } from "enzyme";
import React from "react";
import { AppTopbar } from "./AppTopbar";

describe("test <AppTopbar>", () => {
  let wrapper;
  beforeAll(() => {
    wrapper = mount(<AppTopbar onToggleMenu={e => console.log(e)} />);
  });

  it("renders component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
