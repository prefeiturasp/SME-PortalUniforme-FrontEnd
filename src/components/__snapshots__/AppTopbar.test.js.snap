// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <AppTopbar> renders component 1`] = `
<AppTopbar
  onToggleMenu={[Function]}
>
  <div
    className="layout-topbar clearfix"
  >
    <button
      className="p-link layout-menu-button"
      onClick={[Function]}
    >
      <span
        className="pi pi-bars"
      />
    </button>
    <div
      className="layout-topbar-icons"
    />
  </div>
</AppTopbar>
`;
